spirv_glsl.cpp

```diff
@@ -2787,7 +2811,7 @@ void CompilerGLSL::emit_texture_op(const Instruction &i)
 // Returns the function name for a texture sampling function for the specified image and sampling characteristics.
 // For some subclasses, the function is a method on the specified image.
 string CompilerGLSL::to_function_name(uint32_t, const SPIRType &imgtype, bool is_fetch, bool is_gather, bool is_proj,
-                                      bool has_array_offsets, bool has_offset, bool has_grad, bool has_lod, bool)
+                                      bool has_array_offsets, bool has_offset, bool has_grad, bool has_lod, bool, uint32_t lod)
```

Comment:
Instead of sending in an extra argument, can't has_lod be changed to take the uint32_t value instead?